runners/katacoda/index.ts

```diff
@@ -305,6 +305,32 @@ export class Katacoda extends Runner {
         return null;
     }
 
+    runNextKatacodaStep(step: Step, command: Command): RunResult {
+        let tempFile = path.join(this.getTempDirectory(), command.name + ".md");
+        fs.writeFileSync(tempFile, "");
+        for(let i = 0; i < command.parameters[0].length; i++) {
+            let param = command.parameters[0][i];
+            if(param.content) {
```

Comment:
Wouldn't it be easier to handle the text before and after the image in the asciifile? You wouldn't have to create a new file for each block of text.